nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -873,8 +877,6 @@ define([
             cancelBatch = null;
             updateEditingState();
             switchToTab('configure');
-            // set a new KBase cell ID
-            cell.metadata.kbase.attributes.id = new Uuid(4).format();
```

Comment:
There's a whole load of communication-related machinery (channels, busses, etc.) associated with the cell ID, which was making it inordinately difficult to change the ID. Instead, the cell now sends a `reset-cell` command over the jobCommChannel, which should allow the BE to stop monitoring jobs that are associated with that cell ID.